variables.tf

```diff
@@ -210,6 +210,12 @@ variable "node_use_max_pods" {
   description = "Set to false to prevent EKS from setting --max-pods in Kubelet config. By default, EKS sets the maximum number of pods that can run on the node, based on the instance type. Disabling this can be useful when using a CNI other than the default, like Calico."
 }
 
+variable "show_max_allowed" {
```

Comment:
This variable name does not need to be the same as the script arguemnt, it could be `max_pods_max_allowed` for example.